frontend/src/pages/instance/Settings/ChangeInstanceType.vue

```diff
@@ -1,7 +1,13 @@
 <template>
     <div class="w-full max-w-4xl" data-el="change-project">
         <ff-loading v-if="saving" message="Updating Instance..." />
-        <InstanceForm v-else :instance="instanceDetails || instance" :team="team" :billingEnabled="!!features.billing" @on-submit="changeInstanceDefinition" />
+        <InstanceForm
+            v-else :instance="instanceDetails || instance"
+            :team="team"
+            :billing-enabled="!!features.billing"
+            :flow-blueprints-enabled="!!features.flowBlueprints"
```

Comment:
I don't think in this context (changing an instance type) we ever want to offer the blueprints. I see that is guarded against inside InstanceForm, but would it be more explicit to hardcode this to false here?
```suggestion
            :flow-blueprints-enabled="false"
```